### ISSUE TITLE
Incompatible with ember-auto-import@2.6.0 or higher

### ISSUE THREAD

**Issue by basz** [ember-bootstrap/ember-bootstrap#1865]
On a fairly new project i get this after installing eb in the browser. Compilations seems fine.

anyone seen this?

```
Uncaught Error: Assertion Failed: Expected hash or Mixin instance, got [object Undefined]
    at assert (index.js:153:1)
    at buildMixinsArray (mixin.js:530:1)
    at Mixin.reopen (mixin.js:449:1)
    at DebugFrameworkObject.extend (core.js:372:1)
    at Module.callback (core_view.js:17:1)
    at Module.exports (loader.js:106:1)
    at Module._reify (loader.js:143:1)
    at Module.reify (loader.js:130:1)
    at Module.exports (loader.js:104:1)
    at Module._reify (loader.js:143:1)
```
---

Output from `ember bootstrap:info`:

```
ember bootstrap:info
Npm packages:
ember-bootstrap: ^5.1.1 -> 5.1.1
ember-cli: ^4.9.0 -> 4.10.0
bootstrap: ^5.2.3 -> 5.2.3
bootstrap-sass: n/a
ember-cli-sass: ^11.0.1 -> 11.0.1
ember-cli-less: n/a
Bower packages:
bootstrap: n/a
bootstrap-sass: n/a
ember-bootstrap configuration:
bootstrapVersion: 5
importBootstrapCSS: true
```

**Comment by rogeraraujo90** [comment]
Same issue on my end

**Comment by jelhan** [comment]
Did you tested if it's working with [v6.0.0-1](https://github.com/ember-bootstrap/ember-bootstrap/releases/tag/v6.0.0-1)?

**Comment by rogeraraujo90** [comment]
> Did you tested if it's working with [v6.0.0-1](https://github.com/ember-bootstrap/ember-bootstrap/releases/tag/v6.0.0-1)?

Yep! I did this test. Same result.

**Comment by basz** [comment]
> Did you tested if it's working with [v6.0.0-1](https://github.com/ember-bootstrap/ember-bootstrap/releases/tag/v6.0.0-1)?

yep. same.

**Comment by basz** [comment]
also tried to go back to ember 4.0.0. disabled most other addons. removed most code. could not find a possible culprit.

**Comment by silhusk** [comment]
I encountered the same error in an existing project and could work around it by using `ember-auto-import@2.5.0` instead of 3.x. Not sure if it is the same issue, though.

**Comment by basz** [comment]
> work around it by using `ember-auto-import@2.5.0`

I was using ember-auto-import 2.6.0. Downgrading to 2.5.0 resolves this issue!

**Comment by jelhan** [comment]
Did someone reported the regression to ember-auto-import already?

**Comment by jelhan** [comment]
There is one bug report, which seems related but not including much context: https://github.com/ef4/ember-auto-import/issues/564

**Comment by jelhan** [comment]
This should be fixed by [ember-auto-import@2.6.1`](https://github.com/ef4/ember-auto-import/blob/main/packages/ember-auto-import/CHANGELOG.md#261). I haven't had the chance testing it yet myself.

**Comment by silhusk** [comment]
> This should be fixed by [`ember-auto-import@2.6.1`](https://github.com/ef4/ember-auto-import/blob/main/packages/ember-auto-import/CHANGELOG.md#261). I haven't had the chance testing it yet myself.

Solved for me with `ember-auto-import@2.6.1`.

**Comment by basz** [comment]
solved via [ember-auto-import@2.6.1](https://github.com/ef4/ember-auto-import/blob/main/packages/ember-auto-import/CHANGELOG.md#261)
